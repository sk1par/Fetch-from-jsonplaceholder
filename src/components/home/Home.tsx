
const Home = () => {

  return (
   
<div className="position-relative overflow-hidden p-3 p-md-5 m-md-3 text-center bg-light">
      <div className="col-md-5 p-lg-5 mx-auto my-5">
        <h1 className="display-4 font-weight-normal">Home Page</h1>
        <p className="lead font-weight-normal">And an even wittier subheading to boot. Jumpstart your marketing efforts with this example based on Apple's marketing pages.</p>
        <a className="btn btn-outline-secondary" href="https://www.gong.bg">Coming soon</a>
      </div>
      <div className="product-device box-shadow d-none d-md-block"></div>
      <div className="product-device product-device-2 box-shadow d-none d-md-block"></div>
    </div>
  )
}

export default Home
