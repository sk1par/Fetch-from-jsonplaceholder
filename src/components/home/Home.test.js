import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from '@wojtekmaj/enzyme-adapter-react-17';
import Home from './Home';

Enzyme.configure({ adapter: new Adapter() });

it('should render Home component correctly', () => {
    const wrapper = shallow(<Home />);
    const homeComponentDiv = wrapper.find('[data-auto-id="home-div"]');
    expect(homeComponentDiv).toHaveLength(1);
    expect(wrapper).toMatchSnapshot();
});
