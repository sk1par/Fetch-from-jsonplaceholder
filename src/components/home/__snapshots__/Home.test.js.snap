// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Home component correctly 1`] = `ShallowWrapper {}`;
